src/cargo-kani/src/call_goto_instrument.rs

```diff
@@ -87,6 +89,16 @@ impl KaniSession {
         self.call_goto_instrument(args)
     }
 
+    fn rewrite_back_edges(&self, file: &Path) -> Result<()> {
```

Comment:
Besides the regression failure, could you maybe add basically the pull request description as a doc comment (`///`) on this function? (I know I'm missing those for the others, but let's not make it worse! 😁)